sssd_test_framework/roles/ipa.py

```diff
@@ -672,6 +672,234 @@ def passkey_remove(self, passkey_mapping: str) -> IPAUser:
+
+        return (result, certs)
+
+    def modify_override(
+        self,
+        idview_name: str,
+        *,
+        description: str | None = None,
+        login: str | None = None,
+        uid: int | None = None,
+        gid: int | None = None,
+        gecos: str | None = None,
+        home: str | None = None,
+        shell: str | None = None,
+        sshpubkey: str | None = None,
+        certificate: str | list[str] | None = None,
+    ) -> IDUserOverride:
+        """
+        Modify an User ID override.
+
+        :param idview_name: Name of IDView.
+        :type idview_name: str
+        :param description: Description.
+        :type description: str | None, defaults to None
+        :param login: Overridden User login.
+        :type login: str | None, defaults to None
+        :param uid: Overridden User ID number.
+        :type uid: str | None, defaults to None
+        :param gid: Overridden Group ID.
+        :type gid: str | None, defaults to None
+        :param gecos: Overridden Gecos.
+        :type gecos: str | None, defaults to None
+        :param home: Overridden Home directory.
+        :type home: str | None, defaults to None
+        :param shell: Overridden Login shell.
+        :type shell: str | None, defaults to None
+        :param sshpubkey: Overridden SSH public key.
+        :type sshpubkey: str | None, defaults to None
+        :param certificate: Overridden Certificate.
+        :type certificate: str | list[str] | None, defaults to None
+        :return: self
+        :rtype: IDUserOverride
+        """
+
+        attrs: CLIBuilderArgs = {
+            "desc": (self.cli.option.VALUE, description),
+            "login": (self.cli.option.VALUE, login),
+            "uid": (self.cli.option.VALUE, uid),
+            "gidnumber": (self.cli.option.VALUE, gid),
+            "gecos": (self.cli.option.VALUE, gecos),
+            "homedir": (self.cli.option.VALUE, home),
+            "shell": (self.cli.option.VALUE, shell),
+            "sshpubkey": (self.cli.option.VALUE, sshpubkey),
+            "certificate": (self.cli.option.VALUE, certificate),
+        }
+
+        attrs = CLIBuilderArgs(attrs)
+        self.role.host.conn.exec(
+            ["ipa", "idoverrideuser-mod", idview_name, self.name] + to_list_without_none(self.cli.args(attrs))
+        )
+
+        return self
+
+    def delete_override(self, idview_name: str) -> ProcessResult[ProcessError]:
+        """
+        Delete an User ID override.
+
+        :param idview_name: Name of IDView.
+        :type idview_name: str
+        :return: ProcessResult[ProcessError]
+        :rtype: [ProcessError]
+        """
+
+        result = self.role.host.conn.exec(["ipa", "idoverrideuser-del", idview_name, self.name])
+        return result
+
+    def remove_cert(self, idview_name: str, certificate: str | list[str]) -> IDUserOverride:
+        """
+        Remove one or more certificates to the idoverrideuser entry
```

Comment:
Nitpick, missing period at the end of the sentence.

Reply:
updated.